.circleci/config.yml

```diff
@@ -0,0 +1,158 @@
+version: 2
+
+jobs:
+  checkout: &checkout
+    working_directory: ~/country-template
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - checkout
+
+      - run:
+          name: Fetch remote refs
+          command: git fetch
+
+      - save_cache:
+          key: v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - .
+            - ~/.ssh/known_hosts
+
+  dependencies: &dependencies
+    working_directory: ~/country-template
+    docker:
+      - image: python:2.7.14
+    environment:
+      CREATE_VENV: "virtualenv"
+
+    steps:
+      - restore_cache:
+          keys:
+            - v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+
+      - run:
+          name: Create virtualenv
+          command: |
+            mkdir -p /tmp/venv/country-template
+            ${CREATE_VENV} /tmp/venv/country-template
+
+      # pip >= 8.0 needed to be compatible with "manylinux" wheels, used by numpy >= 1.11
+      - run:
+          name: Install dependencies
+          command: |
+            . /tmp/venv/country-template/bin/activate
+            pip install --upgrade pip twine wheel
+            pip install .[test] --upgrade
+
+      # Uncomment and adapt the next line to use a particular feature branch of OpenFisca-Core to run Circle CI tests
+      # - run:
+      #     name: Install a particular feature branch of OpenFisca-Core
+      #     command: |
+      #       . /tmp/venv/country-template/bin/activate
+      #       pip install --editable git+https://github.com/openfisca/openfisca-core.git@BRANCH_NAME#egg=OpenFisca-Core
+
+      - save_cache:
+          # PY_VERSION is set in the context
```

Comment:
“the context” is ambiguous. I personally don't understand where this is coming from.

Reply:
Got rid of that